sklearn/feature_selection/from_model.py

```diff
@@ -68,11 +71,16 @@ def transform(self, X, threshold=None):
 
         # Retrieve threshold
         if threshold is None:
-            if hasattr(self, "penalty") and self.penalty == "l1":
+            threshold = getattr(self, "threshold", None)
+
+        if threshold is None:
+            # Lasso has a l1 penalty but no penalty param.
+            if (hasattr(self, "penalty") and self.penalty == "l1" or
+                'Lasso' in self.__class__.__name__):
```

Comment:
Hmm. Yes that seems a better idea.

However, sorry if this question seems dumb, but is it worth writing a separate meta-estimator, to just do a masking of the coefs, below a certain threshold value?